samples/dotnet/kernel-syntax-examples/KernelSyntaxExamples.csproj

```diff
@@ -21,6 +21,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="7.0.0" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0-preview.4.23259.5" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="7.0.0" />
```

Comment:
Just checking that we have a .NET 6 project using a 7.0 package and also a 8.0-preview.

We should be fixed to 6.0 only right?
![image](https://github.com/microsoft/semantic-kernel/assets/19890735/e8e8d41a-161b-4332-8c1c-303d3f488a00)

Reply:
It looks like all of those packages support net6.0 by default. The version number isn't in reference to the version of .net it supports.

Reply:
https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection/8.0.0-preview.4.23259.5#supportedframeworks-body-tab